ui/src/components/ActuatorItem.vue

```diff
@@ -0,0 +1,77 @@
+<script setup lang="ts">
+const props = defineProps({
+  name: {
+    type: String,
+    required: true,
+  },
+  data: {
+    type: Object,
+    required: true,
+  },
+});
+
+/**
+ * Convert given bytes to 2 digits precision round exponent version string.
+ * @param bytes number
+ */
+function convertBytes(bytes: number): string {
+  const units = ["bytes", "KB", "MB", "GB", "TB", "EB"];
+  let unitIndex = 0;
+
+  while (bytes >= 1024 && unitIndex < units.length - 1) {
+    bytes /= 1024;
+    unitIndex++;
+  }
+
+  return `${bytes.toFixed(2)} ${units[unitIndex]}`;
+}
+
+/*
+{
```

Comment:
Please remove uncommented code

Reply:
This is structure docs for discussing the template so keep it for now

Reply:
Done